docs/site/Project-layout-reference.md

```diff
@@ -0,0 +1,123 @@
+---
+lang: en
+title: 'Project Layout Reference'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Project-layout-reference.html
+---
+
+LoopBack 4 project files and directories are in the _application root
```

Comment:
We have two types of projects:

- Application project (for LB4 applications)
- Extension project (for LB4 extension modules)

Should we make it explicit this layout is for LB4 applications?

Reply:
Sure, I can change the title as well if we need another page for extension project.